components/prism-stylus.js

```diff
@@ -0,0 +1,61 @@
+Prism.languages.stylus = {
+	'comment': {
+		pattern: /(^|[^\\])(\/\*[\w\W]*?\*\/|\/\/.*?(\r?\n|$))/g,
+		lookbehind: true
+	},
+	'keyword': /(px|r?em)\b|\b(is|defined|not|isnt|and|or|unless|for|in)\b/g,
```

Comment:
What about all the other CSS units? (http://www.w3.org/TR/css-values/#lengths chapters 5 and 6)